core/src/test/java/smile/classification/NaiveBayesTest.java

```diff
@@ -307,7 +385,7 @@ public void testLearnBernoulli2() {
             }
         }
 
-        System.out.format("Bernoulli error = %d of %d%n", error, total);
```

Comment:
same above.

Reply:
Yes I will keep it as it was. Thanks for pointing out